client/asset/btc/spv.go

```diff
@@ -144,7 +144,7 @@ func createSPVWallet(privPass []byte, seed []byte, dbDir string, log dex.Logger,
 
 	pubPass := []byte(wallet.InsecurePubPassphrase)
 
-	_, err = loader.CreateNewWallet(pubPass, privPass, seed, walletBirthday)
+	_, err = loader.CreateNewWallet(pubPass, privPass, seed, bday)
```

Comment:
We need to consider what to do about/with the existing `spvWallet.birthday` field.

Here on initial wallet **creation** the passed birthday gets stored in the wallet Manager DB, and then on wallet startup (`startWallet`) it loads it like `spvWallet.birthday = btcw.Manager.Birthday()`.  This field was just used in the syncStatus method so far.

If someone later opens the wallet (or reconfigures it) with a different wallet birthday in the settings, `openSPVWallet` will record that in `ExchangeWalletSPV.walletBirthday`, but it does not seem to update the field in `spvWallet` or in the Manager's DB (unless rescan is done in the case of the DB).

Maybe I'm not seeing it all clearly yet, but I think we need to look at merging the two bday fields.  It does make sense though not to update the wallet db until a rescan has actually been applied with the new birthday.

Reply:
It's definitely incorrect that `spvWallet.birthday` was not getting updated during the rescan, I've updated that now. The way I see it currently is that `spvWallet.birthday` is basically a cache for what is stored in the wallet db, both of which are updated when a rescan is done, and `ExchangeWalletSPV.walletBirthday` represents the latest selection by the user. I think that's fairly reasonable.. what's the reason they should be merged? Should we automatically start a rescan if the birthday is updated?

Reply:
> The way I see it currently is that `spvWallet.birthday` is a cache for what is stored in the wallet db, both of which are updated when a rescan is done, and `ExchangeWalletSPV.walletBirthday` represents the latest selection by the user. I think that's fairly reasonable.. what's the reason they should be merged?

That's a reasonable interpretation.  It was just that two birthday fields, one at high level and one low, stood out as a possible mistake.  But how I would answer your question is that there's a likelihood of them diverging accidentally, as happened in the initial draft.  We can run with what you've got now though IMO.